SocialFilter/sections/css_extended.txt

```diff
@@ -40,6 +40,7 @@ thepythoncode.com#?#div[align="center"]:has(> span:contains(/^Sharing is caring/
 verstka.media#?#main em + a[href="https://t.me/svobodnieslova"]:upward(1)
 verstka.media#?#main div[class^="wp-container-"]:has(> h2.verstka-telegram-link:not([style]))
 rlx.jp#?#.whiteBg > div.sectionBox:has(> div.inner > div.shearBox)
+mbs.jp#?#.foot-sns > div[class^="foot-sns_inner"]:upward(1)
```

Comment:
Why not `##.foot-sns`? False positive on other page?

Reply:
I thought `##.foot-sns` is a general rule to may cause problems on other webpages.
Do I worry too much about it?

Reply:
I mean `mbs.jp##.foot-sns`

Reply:
yeah, I'm also talking about it.